src/cephadm/cephadm

```diff
@@ -8326,7 +8460,7 @@ def _get_parser():
     parser_bootstrap.add_argument(
         '--skip-monitoring-stack',
         action='store_true',
-        help='Do not automatically provision monitoring stack (prometheus, grafana, alertmanager, node-exporter)')
+        help='Do not automatically provision monitoring stack (prometheus, grafana, alertmanager, node-exporter, loki, promtail)')
```

Comment:
Wouldn't it make sense to allow users still deploy the current monitoring stack without the centralized logging, and require them to add an extra flag for it (e.g.: `--centralized-logging`)? I'm asking this because the extra resources required (central loki + per-host promtail containers) are not negligible.

Reply:
We are not deploying the loki and promtail services by default. The user can create the services if he wants to use the centralised logging